src/libsync/capabilities.cpp

```diff
@@ -222,6 +222,12 @@ bool Capabilities::versioningEnabled() const
     return _capabilities.value(QStringLiteral("files")).toMap().value(QStringLiteral("versioning")).toBool();
 }
 
+bool Capabilities::avatarsAvailable() const
+{
+    // true by default for older servers, because turning off profile pictures was introduced in later versions
+    return _fileSharingPublicCapabilities.value(QStringLiteral("profile_picture"), true).toBool();
```

Comment:
I think that should be `_capabilities.value("user").toMap().value(`
Please retest with demo.owncloud.com

Reply:
no, apparently it's `files_sharing` -> `user` -> `profile_picture`.

I thought it was on the same level as `expire_date`, but it turns out there are 4 places where `expire_date` occurs.